### DISCUSSION TITLE
PrimeNg v18 incompatible with Angular 19

### DISCUSSION THREAD

**Discussion by Luperco** [primefaces/community#3178] [PrimeNG]
Hi,
when I try to install the last version of PrimeNg (v18) that is supposed to be compatible with Angular 19, I receive this error:

error peer @angular/animations@"^17.0.0 || ^18.0.0" from primeng@18.0.0

**Comment by photon0001** [comment]
Open issue [16827](https://github.com/primefaces/primeng/issues/16827)
